README.md

```diff
@@ -178,6 +179,40 @@ end
 
 For more detailed usage, please check the instrumentation's page.
 
+## Grape
+
+This instrumentation subscribes to ActiveSupport notifications emitted by the Grape API.
+
+The source for this instrumentation is located [here](https://github.com/signalfx/ruby-grape-instrumentation)
+
+### Usage
+
+```ruby
+SignalFx::Tracing::Instrumenter.configure do |p|
+    p.instrument(:Grape)
+end
+```
+
+`instrument` takes two optional arguments:
+- `tracer`: custom tracer to use. Defaults to `OpenTracing.global_tracer`
+- `parent_span`: parent span to group spans or block that returns a span. Default: `nil`
+
+Other than `parent_span`, spans can also be grouped by Rack request.
+
+In order to use the Rack instrumentation and group spans by request, you must
```

Comment:
I'm not sure it's very clear what situation this covers. I'm guessing this only concerns people that use Grape with Rack. If they are in that situation, why wouldn't they "group spans by request" (whatever that means)? If that's the better default behavior, is it possible for us to detect if we're in this situation and insert the middleware for them?

Reply:
The detection is a bit messy due to API changes between versions and the class handling mechanism in the library, which is why I excluded it at first.

I think we can maybe patch by default, and have the option to disable it if it causes issues.